### DISCUSSION TITLE
How to set flat accel_profile for mouse, and adaptive for touchpad?

### DISCUSSION THREAD

**Discussion by Kommynct** [hyprwm/Hyprland#5580] [Q&A]
I've tried

```
input {
	accel_profile = flat
	touchpad {
		accel_profile = adaptive
	}
}
```

But that doesn't seem to work

**Comment by Gliczy** [comment]
Take a look at https://wiki.hyprland.org/Configuring/Keywords/#per-device-input-configs

**Reply by Kommynct** [reply]
I did look at that, however, that is for specific devices, I want ALL mice to be flat and ALL touchpads to be adaptive
